README.md

```diff
@@ -38,6 +38,37 @@ python3 -m pip install --no-build-isolation git+https://github.com/scientificcom
 
 Note that you should pass the flag `--no-build-isolation` to `pip` to avoid issues with the build environment, such as incompatible versions of `nanobind`.
 
+### `spack`
+The spack package manager is the recommended way to install scifem, and especially on HPC systems.
+For information about the package see: [spack-package: py-scifem](https://packages.spack.io/package.html?name=py-scifem)
+First, clone the spack repository and enable spack
+
+```bash
+git clone --depth=2 https://github.com/spack/spack.git
+# For bash/zsh/sh
+. spack/share/spack/setup-env.sh
+
+# For tcsh/csh
+source spack/share/spack/setup-env.csh
+
+# For fish
+. spack/share/spack/setup-env.fish
+```
+
+Next create an environment:
+
+```bash
+spack env create scifem_env
+spack env activate scifem_env
+```
+
+and install the relevant packages
+```bash
+spack add py-scifem+petsc+hdf5+biomed+adios2 ^petsc+mumps+hypre ^py-fenics-dolfinx+petsc4py
+spack concretize
+spack install
+```
+
```

Comment:
Finally, note that spack needs some packages already installed on your system. On a clear ubuntu container for example one need to install the following packages before running spack
```bash
apt update && apt install gcc unzip git python3-dev g++ gfortran xz-utils -y
```